blocks/bricks/recurrent.py

```diff
@@ -32,6 +32,10 @@ class BaseRecurrent(Brick):
     def initial_state(self, state_name, batch_size, *args, **kwargs):
         r"""Return an initial state for an application call.
 
+        Default implementation returns a zero matrix. All the standard
+        recurrent bricks override it with trainable initial states
```

Comment:
"standard recurrent bricks" is a bit vague, I guess you mean bricks derived from `SimpleRecurrent`?